kvmagent/kvmagent/plugins/host_plugin.py

```
@@ -2974,26 +2961,35 @@ def set_ip_on_host_network_interface(self, req):
+                    # zs-network-setting -i eth0 192.168.1.10 255.255.255.0
+                    # 192.168.1.1
+                    if cmd.gateway is not None:
+                        shell.call('/usr/local/bin/zs-network-setting -i %s %s %s %s' %
+                                   (cmd.interfaceName, cmd.ipAddress, cmd.netmask, cmd.gateway))
+                    else:
+                        # zs-network-setting -d eth0
+                        shell.call('/usr/local/bin/zs-network-setting -d %s' %
+                                   cmd.interfaceName)
+                        bash_o('/usr/local/bin/zs-network-setting -i %s %s %s' %
+                               (cmd.interfaceName, cmd.ipAddress, cmd.netmask))
             except Exception as e:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)